### PR TITLE
[BUGFIX] Pause Menu and Stickers have same zoom as HUD

### PR DESCRIPTION
**Pull request by gamerbross**
(https://github.com/FunkinCrew/funkin.assets/pull/7 required!!!)
The Pause Menu and Stickers Transition were using camHUD, which its zoom can change, scaling these without having to.
I added `camOther` as another camera which contains video cutscenes, dialogues, pause menu and sticker transition (or whatever that shouldn't have the same zoom as HUD)
I also removed `camCutscene`, because can be replaced with this new one.

Here I'm going to attach some issues caused by unintentional zoom.
(M.I.L.F. is a great song to use, as it zooms the HUD more)
---
image
_Pause Menu getting cut by the HUD zoom._

https://github.com/FunkinCrew/Funkin/assets/55158797/dc49a9e9-ed41-43e2-bf7a-a2d2b841d855

_Sticker Transition zooming out a little bit_

### CODE REVIEW

**Comment by Matriculaso** [comment]
Delete this shit bro

**Comment by Matriculaso** [comment]
Pretty Useful tbh, I had an issue with stickers cuz I used a lot of camZoomings events and stickers were pretty messed up along with pause menu, Thanks Gamerbross!

**Comment by gamerbross** [comment]
Ok it has some issues that I didn't saw because when I was trying it, camCutscene still existed, so there were no problems

**Comment by gamerbross** [comment]
Just fixed those issues (now needs this pr in funkin.assets https://github.com/FunkinCrew/funkin.assets/pull/7)
(it also fixed one bug in the roses end cutscene!!!)
image
Pause menu was on camHUD, so this red camera flash on camCutscene covered it, putting everything red.

**Comment by Matriculaso** [comment]
hurry up and add this shit my grilfriend feet sticker pack needs this!!! feet is too big because of the zoom I do on her feet helpp pleass 😫😫😫😫

**Comment by Stilic** [comment]
the renaming isn't needed.
it could cause breaking with existing scripts.

File: source/funkin/play/PlayState.hx [hunk 1]
```diff
@@ -491,9 +491,9 @@ class PlayState extends MusicBeatSubState
   public var camGame:FlxCamera;
 
   /**
-   * The camera which contains, and controls visibility of, a video cutscene.
+   * The camera which contains, and controls visibility of, a video cutscene, dialogue, pause menu and sticker transition.
    */
-  public var camCutscene:FlxCamera;
+  public var camOther:FlxCamera;
```
Comment (DMMaster636) [inline source/funkin/play/PlayState.hx:496]: Why replace camCutscene?? That can lead to some broken scripts.
Wouldn't it be smarter to just add a new camera for this?

**Review by DMMaster636 — CHANGES_REQUESTED** [review]

File: source/funkin/play/PlayState.hx (hunk 1 above)
Comment (EliteMasterEric) [reply source/funkin/play/PlayState.hx:496]: That's true, it also breaks a bunch of cutscene scripts and those would need to be redone.

Comment (gamerbross) [reply source/funkin/play/PlayState.hx]: I renamed it because it didn't just handle cutscenes now, but I just reverted it

**Comment by gamerbross** [comment]
I reverted the camCutscene rename, so now the pull request in assets isn't really required, just removes unnecessary visibility changes to the camera

**Comment by Matriculaso** [comment]
image

**Review by EliteMasterEric — APPROVED** [review]
Works good for me

**Review by Cartridge-Man — APPROVED** [review]

**Pull request merged**